src/Recordings.cpp

```diff
@@ -568,7 +568,12 @@ PVR_ERROR Recordings::SetRecordingLastPlayedPosition(const kodi::addon::PVRRecor
     time_t timerUpdate = m_timers.m_lastTimerUpdateTime;
     if (lastplayedposition == -1)
     {
-      lastplayedposition = recording.GetDuration();
+      if (recording.GetRecordingTime() + recording.GetDuration() > time(nullptr)) {
+        lastplayedposition = time(nullptr) - recording.GetRecordingTime() - 5;
+        isWatched = false;
+      }
+      else
+        lastplayedposition = recording.GetDuration();
```

Comment:
Generally if you use brackets for the if then you should for the else too.

Reply:
Ok I often only do braces when there are more than one line but I will change that.